[2023-02-18T18:29:07.913-0600] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1.5_user_processing.is_api_available scheduled__2023-02-18T00:00:00+00:00 [queued]>
[2023-02-18T18:29:07.925-0600] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1.5_user_processing.is_api_available scheduled__2023-02-18T00:00:00+00:00 [queued]>
[2023-02-18T18:29:07.926-0600] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-02-18T18:29:07.927-0600] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-02-18T18:29:07.927-0600] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-02-18T18:29:07.938-0600] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-02-18 00:00:00+00:00
[2023-02-18T18:29:07.941-0600] {standard_task_runner.py:54} INFO - Started process 196101 to run task
[2023-02-18T18:29:07.943-0600] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '1.5_user_processing', 'is_api_available', 'scheduled__2023-02-18T00:00:00+00:00', '--job-id', '512', '--raw', '--subdir', 'DAGS_FOLDER/Introduction_to_Apache_Airflow/1.5_user_processing.py', '--cfg-path', '/tmp/tmpou4yiqll']
[2023-02-18T18:29:07.946-0600] {standard_task_runner.py:83} INFO - Job 512: Subtask is_api_available
[2023-02-18T18:29:07.947-0600] {dagbag.py:525} INFO - Filling up the DagBag from /c/Airflow/dags/Introduction_to_Apache_Airflow/1.5_user_processing.py
[2023-02-18T18:29:08.269-0600] {task_command.py:384} INFO - Running <TaskInstance: 1.5_user_processing.is_api_available scheduled__2023-02-18T00:00:00+00:00 [running]> on host Tinmar.
[2023-02-18T18:29:08.336-0600] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Tinmar
AIRFLOW_CTX_DAG_ID=1.5_user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-02-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-18T00:00:00+00:00
[2023-02-18T18:29:08.339-0600] {http.py:122} INFO - Poking: api/
[2023-02-18T18:29:08.355-0600] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-02-18T18:29:08.770-0600] {base.py:213} INFO - Success criteria met. Exiting.
[2023-02-18T18:29:08.781-0600] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=1.5_user_processing, task_id=is_api_available, execution_date=20230218T000000, start_date=20230219T002907, end_date=20230219T002908
[2023-02-18T18:29:08.797-0600] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-02-18T18:29:08.822-0600] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
